airflow/contrib/executors/kubernetes_executor.py

```diff
@@ -123,6 +123,10 @@ def __init__(self):
         self.core_configuration = configuration_dict['core']
         self.kube_secrets = configuration_dict.get('kubernetes_secrets', {})
         self.kube_env_vars = configuration_dict.get('kubernetes_environment_variables', {})
+        self.kube_env_from_configmap_ref = configuration.get(self.kubernetes_section,
+                                                             'env_from_configmap_ref')
+        self.kube_env_from_secret_ref = configuration.get(self.kubernetes_section,
+                                                          'env_from_secret_ref')
```

Comment:
To be consistent with #4772 I would suggest using `self.kube_secrets` and add `self.kube_configmaps` .